### ISSUE TITLE
Draw Arrow shape error of label

### ISSUE THREAD

**Issue by ychyss** [ivmartel/dwv#1441]
This problem can be reproduced in the `dwv/pacs/viewer.js` example at version 0.32.0

1.  open `dwv/pacs/viewer.js` and add shapes of Draw `options: ['Ruler', 'Rectangle', 'Ellipse', 'Arrow']` in function `viewerSetup`, so I can draw more shapes.
2. change the shapeName in function `setupToolsCheckboxes` from `Rectangle` to `Arrow`.
3.  open files, and select Draw tool, draw an arrow
4. edit it to add a label then got error of `group.findOne is not a function`

The stack error message is here:
![image](https://github.com/ivmartel/dwv/assets/15881179/2902d9d9-f4e6-45fa-b8e9-b677ccd4ca28)

**Comment by ychyss** [comment]
I tracked the bug and found that when the shape is created, it will enter `this.#onFinalPoints` in `src/tools/draw.js`. First, it will delete `this.#tmpShapeGroup` (assuming ID 174), then create the shape `finalShapeGroup` (assuming ID 180), and finally enter `setShapeOn` function to set event associations.

But when I double clicked the arrow in viewer, I found that the `event.target` obtained in dblclick handle was not the correct value (assuming ID 175 and it don't have findOne function, I don't know which class of instance it is). I am not very sure why this is, and the ID is also not `tmpShapeGroup`. But when I check `event.currentTarget`, this is the correct value(ID is 180). Of course, it can also be directly changed to `shapeGroup`(maybe not good).

And  what confused me was that I also tracked the handles to the `dragmove.draw` and `dragent.draw` events, as they both called `event.target`, but they could both obtain the correct objects. So I wonder if there is a difference in Konva's logic new version(8.4.2)? Or did other generated objects cause an error in event triggering?
